### ISSUE TITLE
Options to choose writer or reader endpoints on aurora RDS

### ISSUE THREAD

**Issue by koneru9999** [BohdanPetryshyn/basti#34]
At the moment, basti presents all the RDS instances irrespective of whether it is for an aurora cluster or nor normal RDS instance. 

It would be great to present just the writer / reader endpoints especially where there are many readers in a aurora cluster.

**Comment by BohdanPetryshyn** [comment]
@koneru9999, thank you for your feedback!

As far as I understand, this issue requests two changes:
1. Add the ability to select between writer and reader endpoints when connecting to RDS/Aurora clusters
2. Filter out instances that are part of RDS/Aurora clusters and display only "independent" RDS instances in the "Database Instances" section in interactive mode

Please, correct me if I'm wrong 🙏

**Comment by koneru9999** [comment]
@BohdanPetryshyn that is correct and exactly what i am looking for. Sorry for not being explicit earlier.

**Comment by BohdanPetryshyn** [comment]
Both aspects look reasonable to me and will be implemented soon 😉

Thank you again for your feedback, @koneru9999!
